### PR TITLE
Use path mapping for scoped packages

### PR DESCRIPTION
**Pull request by ghost**
Depends on Microsoft/types-publisher#356

### CODE REVIEW

File: types/storybook__addon-knobs/tsconfig.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
+            "dom",
```
Comment (joscha) [inline types/storybook__addon-knobs/tsconfig.json]: 💯 ta!

**Review by joscha — APPROVED** [review]
Changes in all `storybook__*` look good to me, thanks for following up on this!

**Comment by sandersn** [comment]
Travis failed; something about "unexpected path mapping".

**Comment by andy-ms** [comment]
Waiting on Microsoft/types-publisher#356.

**Comment by sandersn** [comment]
Ah, I missed that.

**Comment by joscha** [comment]
@andy-ms could you add the changes in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/17498 once @sandersn merged it to this PR, please? If that's not desirable I can open a follow-up once this one is merged.

**Comment by RyanCavanaugh** [comment]
@andy-ms please fix

**Pull request merged**